Tests/PennyTests/GHHooksTests.swift

```diff
@@ -17,10 +18,16 @@ class GHHooksTests: XCTestCase {
         return decoder
     }()
 
+    let httpClient = HTTPClient(eventLoopGroupProvider: .createNew)
+
     override func setUp() async throws {
         FakeResponseStorage.shared = FakeResponseStorage()
     }
 
+    override func tearDown() {
+        try! httpClient.syncShutdown()
```

Comment:
_i think_ that was intentional by me to do it the try! way, just so i don't have to use the async version of the shutdown method, because if you change the signature to async-throws, i think it'll emit warnings and ask you to use the async version.